src/components/SubNavigationMenu/SubNavigationMenu.tsx

```diff
@@ -0,0 +1,196 @@
+/** @jsxImportSource theme-ui */
+import { ReactElement, MouseEventHandler, useState, cloneElement } from "react"
+import { ThemeUICSSObject } from "theme-ui"
+import Button from "../Button/Button"
+import { SubNavigationMenuContext, useSubNavigationMenu } from "./Context"
+
+interface SubNavigationMenuItemProps {
+  children?: ReactElement
+  label: string
```

Comment:
Is the `label` prop really needed if `children` is used? Is there any way we can merge these two into one prop?